AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -66,6 +71,8 @@ - (instancetype)init {
   if ((self = [super init])) {
```

Comment:
pass `MS_Reachability` as and argument so that you can mock it in unit test

Reply:
I think we already did that? :)

Reply:
in the test case ```  // Simulate being offline.
  MS_Reachability *reachabilityMock = OCMPartialMock([MS_Reachability reachabilityForInternetConnection]);
  OCMStub([reachabilityMock currentReachabilityStatus]).andReturn(NotReachable);
  self.sut.reachability = reachabilityMock;```

Reply:
it works as you implemented, yes it should not be a public setter. I was suggesting to pass is as constructor argument

Reply:
It's more like an internal setter because it isn't in the public header file